app-editors/featherpad/featherpad-0.18.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg qmake-utils
+
+MY_P="${P/featherpad/FeatherPad}"
+
+DESCRIPTION="Lightweight Qt5 Plain-Text Editor for Linux"
+HOMEPAGE="https://github.com/tsujan/FeatherPad"
+SRC_URI="https://github.com/tsujan/FeatherPad/archive/V${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="app-text/hunspell
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtx11extras:5"
+DEPEND="${RDEPEND}
+	dev-qt/linguist-tools:5"
+
+S="${WORKDIR}/${MY_P}"
+
+src_configure() {
+	eqmake5
+}
```

Comment:
I have a feeling this should be `src_compile` phase instead.

Reply:
This one's fine in src_configure, it's doing Makefile configuration to be run by normal emake in src_compile/install

That aside, i see this package has a CMakeLists.txt (unsure if it's in a working state) -- if cmake is usable it'd be better than the qmake mess.

Reply:
Thx for checking, but qmake just compiles the Qt project file and generates the Makefile that is used during `the src_compile `phase.

Reply:
Good to know, never wrote a qmake ebuild - it just looked like an alternative to `emake` instead of `econf` :)